src/Builder/Encoder/OperatorEncoder.php

```diff
@@ -55,7 +46,8 @@ private function encodeAsArray(OperatorInterface $value): stdClass
     {
         $result = [];
         /** @var mixed $val */
-        foreach (get_object_vars($value) as $val) {
+        foreach ($value::PROPERTIES as $prop => $name) {
```

Comment:
Loops over existing array instead of converting object to array with `get_object_vars`. Must be more performant.